### ISSUE TITLE
Table a11y issues

### ISSUE THREAD

**Issue by jschuler** [patternfly/patternfly-react#5099]
https://github.com/patternfly/patternfly-react/pull/4950#pullrequestreview-520904212

>- Headers repeat in VO (not quite sure what is causing this, ex: "Workspaces, Workspaces column 4 of 5"), it doesn't make it inaccessible by any means but could be annoying to screen reader users.
>- All truncated tooltips should have `tabindex="0"` so that keyboard users can also see truncated text.
>- The sortable button could use a label to let screen reader users know what this button does.
>- Testing this makes me question how we can shift screen reader user focus to a popover when it's opened. It's connected to click events and currently when using a popover we don't shift focus to the popover when it's clicked. When popovers are used in the table, I don't see them in VO at all.
>- It does pass an html validator, though it does have one error in terms of the `aria-describedby`. It looks like it's pointing to something that I can't find in the DOM? I assume it's pointing to the tooltip before it exists?
